app/domain/authentication/o_auth/discover_identity_provider.rb

```diff
@@ -34,7 +34,7 @@ def discover_provider
         @discovered_provider = @open_id_discovery_service.discover!(@provider_uri).tap do
           @logger.debug(Log::IdentityProviderDiscoverySuccess.new)
         end
-      rescue HTTPClient::ConnectTimeoutError => e
+      rescue HTTPClient::ConnectTimeoutError, Errno::ETIMEDOUT => e
```

Comment:
notice that also here, https://github.com/lostisland/faraday/issues/718
they use ::HTTPClient::TimeoutError, Errno::ETIMEDOUT

Seems that this fixes the issue but I would like to understand the difference between these 2.